app/policies/comment_policy.rb

```diff
@@ -0,0 +1,29 @@
+class CommentPolicy < ApplicationPolicy
+  authorize :user, allow_nil: true
```

Comment:
Why we need it?

Reply:
This line allows the current user to be empty. The default value for "user" does not depend on this.

Reply:
so we need the user to be always present?

Reply:
For our case yes